lib/defaults.go

```diff
@@ -0,0 +1,14 @@
+package lib
+
+import (
+	"runtime"
+)
+
+// RootUser returns the name of the "root" / Admin user
+func RootUser() string {
+	if runtime.GOOS == "windows" {
+		return "Administrator"
```

Comment:
If you wanted, you could do this at buildtime with a `defaults_unix.go` and `default_windows.go` that define the same constant.